app/controllers/invites_controller.rb

```diff
@@ -40,9 +40,16 @@ def invite_params
   end
 
   def set_user_to_id
-    @user_to = User.find_by(email: invite_params[:email]).id
-  rescue NoMethodError
-    redirect_to account_invites_url, notice: 'Field should\'t be blank'
+    email = invite_params[:email]
+    if email.blank?
+      redirect_to account_invites_url, notice: 'Field should\'t be blank'
+    else
+      begin
+        @user_to = User.find_by(email: email).id
```

Comment:
Using exceptions as system workflow control is not a good practice, actually I supposed you to check if user found and than get it's `id`